src/Toml/BiMap.hs

```diff
@@ -96,41 +94,52 @@ _Right = invert $ prism (either (const Nothing) Just) Right
 -- | Creates prism for 'AnyValue'.
 mkAnyValueBiMap :: (forall t . Value t -> Maybe a)
                 -> (a -> Value tag)
-                -> BiMap AnyValue a
+                -> BiMap a AnyValue
 mkAnyValueBiMap matchValue toValue =
-    prism (\(AnyValue value) -> matchValue value) (AnyValue . toValue)
+    BiMap (Just . AnyValue . toValue) (\(AnyValue value) -> matchValue value)
+
+-- | Creates prism for 'Text' to 'AnyValue' with custom functions
+_TextBy :: (Text -> Maybe a) -> (a -> Text) -> BiMap a AnyValue
```

Comment:
Wasn't that function implemented in different PR? I don't see in the deleted diff...

Reply:
Yes, it's a mistake on my part. It was added in #104 which was merged after I submitted this PR.  But I messed up my branches and ended up including it here. I'll make sure it merges properly.